setup.cfg

```diff
@@ -1,3 +0,0 @@
-[bdist_wheel]
```

Comment:
Why delete this file? Was it not being used?

Reply:
It was not being used. 

> Thanks!
> Are links added to the readme that point to the new scripts....I don't see them in the diff.

I've updated the readme for the new way to build. I didn't add the make wheel to the README because it's really a testing automation workflow and not instructions to help people build and test coremltools.

Reply:
@aseemw to clarify I split the long readme into a README for users and a BUILD.md to make it clear that those 2 are separate. Sometimes users can get intimated by long readme thinking they need that to build and use the package.